packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.ts

```diff
@@ -292,7 +321,7 @@ export class AtomicCommerceProductList
   }
 
   private computeListDisplayClasses() {
-    const displayPlaceholders = !this.isAppLoaded;
+    const displayPlaceholders = !(this.isAppLoaded && this.isProductsReady);
```

Comment:
We could put this in a private getter and reuse it in the `when` directive in the render function.